core/basic_models/actions/basic_actions.py

```diff
@@ -52,7 +52,7 @@ class CommandAction(Action):
     version: Optional[int]
     command: str
     request_type: Optional[str]
-    request_data: Optional[Dict]
+    request_data: Optional[Dict[str, str]]
```

Comment:
```suggestion
    request_data: Optional[Dict[str, Any]]
```
Нашёл во Фрейде такой usage: answer.request_data['app_callback_id'].encode(), где encode - метод bytes.